src/app/components/attemp-rows/attemp-rows.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input } from '@angular/core';
+import { Attempt } from 'src/app/interfaces/attempt.interface';
+import { Letter } from 'src/app/interfaces/letter-status.interface';
+
+@Component({
+  selector: 'app-attemp-rows',
+  templateUrl: './attemp-rows.component.html',
+  styleUrls: ['./attemp-rows.component.scss'],
+})
+export class AttempRowsComponent {
+  @Input() attempsRows!: Attempt[];
```

Comment:
```suggestion
  @Input() attempsRows: Attempt[] = [];
```

Its better to assing a default value instead telling que compiler that this Input has going to have value.